scalameta/transversers/shared/src/test/scala/meta/transversers/OrderTests.scala

```diff
@@ -0,0 +1,109 @@
+package scala.meta.transversers
```

Comment:
Recently we have moved the overwhelming majority of tests into a single big project at `scalameta/tests`. Let's continue doing this in future.